### PR TITLE
Update _index.md with todo list

### PR DESCRIPTION
**Pull request by AntonioDiTuri**

### CODE REVIEW

File: website/content/cloud-native-sustainability-week/_index.md [hunk 1]
```diff
@@ -63,6 +63,12 @@ In addition to the global virtual event organized by the CNCF TAG ENV, there wil
 | 23 | USA, Seattle | *tbd* | *tbd* | Yiming Peng, Kate Goldenring |
 | 24 | USA, South Florida | Oct.19 | [Link](https://community.cncf.io/events/details/cncf-south-florida-presents-south-florida-meetup-cncf-cloud-native-sustainability-week/) | Angel Ramirez, Hitomi Mizugaki, Cristher Castro |
 
+#### To-do list for local MeetUp organizer
+
+Here you can find an helpful todo-list for everything you need to do to and know before-while-and-after the meetup:
```
Comment (leonardpahlke) [inline website/content/cloud-native-sustainability-week/_index.md:68]:
```suggestion
All things you should know about as local meetup organizer before, during and after the event are documented in this [issue](https://github.com/cncf/tag-env-sustainability/issues/224).
```

File: website/content/cloud-native-sustainability-week/_index.md [hunk 2]
```diff
@@ -63,6 +63,12 @@ In addition to the global virtual event organized by the CNCF TAG ENV, there wil
...
+
+[To do list](https://github.com/cncf/tag-env-sustainability/issues/224)
```
Comment (leonardpahlke) [inline website/content/cloud-native-sustainability-week/_index.md:70]:
```suggestion
```

**Comment by leonardpahlke** [comment]
Thanks @AntonioDiTuri

**Comment by guidemetothemoon** [comment]
@AntonioDiTuri @leonardpahlke since we shared checklist in a different way and are now closing in on the end of the Cloud Native Sustainability Week do you think that this checklist is still relevant to be included in our website? I think it can potentially be useful to have it publicly available, maybe it can come in handy for those who will be organizing meetups in the coming weeks and also for future reference, for any future Cloud Native Sustainability Week events we may decide to organize. WDYT?

**Comment by leonardpahlke** [comment]
We can close this PR, thanks @AntonioDiTuri @guidemetothemoon

**Pull request closed without merging**